### DISCUSSION TITLE
Slack command payload

### DISCUSSION THREAD

**Discussion by Foldager94** [Churnr/churnr-crm#26] [Show and tell]
Hej @bentreadwell og @groenborg.

Når slack command sender et request med parameter til vores backend, kommer alle parameterne i samme felt under body, i et felt der hedder text. Det er den nuværende løsning vi har saat op, men der er muligvis en måde hvor på man kan gøre det mere strics på. Ville det være en ide at kigge ind i dette, eller skal createCustomer forgå et andet sted end på slack i fremtiden?

Dette er den nuværende command template for createCustomer, som lidt er noget rod:

/createcustomer companyName="companyNameTest", paymentGateway="paymentGatewayTest", apiKey="apiKeyTest", emailGateway="emailGatewayTest", emailGatewayUser="emailGatewayUserTest", emailGatewayPassword="emailGatewayPasswordTest", contactPerson="contactPersonTest", flowEmails="flowEmailsTest", flowCalls="flowCallsTest"

Evt. noget vi kan tage til fredags mødet

**Comment by groenborg** [comment]
Hey @Foldager94 

Lad os snakke om den her i løbet af næste uge. Vi kan lave slack templates som kan hjælpe med det her input og gøre det mere spiseligt for benjamin. Ligeledes vil jeg også mene at vi skal kigge på noget naming. Jeg kan gennemskue er der er tekniske detailjer i de her navne. Det skal en bruge aldrig vide noget om. Navnene på de felter de skal udfylde skal være sigende og ikke til at tage fejl af.

**Reply by Foldager94** [reply]
Hej @groenborg 

Det kunne være fedt hvis vi lige kunne se på det sammen. Vi kunne evt. slå det sammen med den dag hvor vi kigger på koden?
